examples/ucx/local_cuda_cluster.py

```diff
@@ -0,0 +1,59 @@
+import click
+
+from dask.utils import parse_bytes
+
+from dask_cuda import LocalCUDACluster
+
+
+@click.command(context_settings=dict(ignore_unknown_options=True))
+@click.option(
+    "--enable-nvlink/--disable-nvlink",
+    default=False,
+    help="Enable NVLink communication",
+)
+@click.option(
+    "--enable-infiniband/--disable-infiniband",
+    default=False,
+    help="Enable InfiniBand communication with RDMA",
+)
+@click.option(
+    "--interface",
+    default=None,
+    type=str,
+    help="Enable InfiniBand communication with RDMA",
+)
+@click.option(
+    "--rmm-pool-size",
+    default="1GB",
+    type=parse_bytes,
+    help="If specified, initialize each worker with an RMM pool of "
+    "the given size, otherwise no RMM pool is created. This can be "
+    "an integer (bytes) or string (like 5GB or 5000M).",
+)
+def main(
+    enable_nvlink, enable_infiniband, interface, rmm_pool_size,
+):
+
+    enable_rdmacm = False
+    ucx_net_devices = None
+
+    if enable_infiniband:
+        enable_rdmacm = True
+        ucx_net_devices = "auto"
+
+    if (enable_infiniband or enable_nvlink) and not interface:
+        raise ValueError("Interface must be specified if using NVLink or Infiniband")
+
+    cluster = LocalCUDACluster(  # noqa F842
```

Comment:
How can we keep this cluster running until the user manually terminates the script? I would like it so a user can use this script to start up a cluster that can later be connected to by the example client initialization script.

Reply:
`LocalCUDACluster` is generally intended to be used in the same process as `Client`, and in that case the `Client` doesn't need to set additional variables as `LocalCUDACluster` already took care of that. With that in mind, I would simply suggest we add:

```python
from distributed import Client

...

client = Client(cluster)
```

Doing the above would make this sample complete IMO.